structuredefinitions/UKCore-Observation-Group-Lab.xml

```diff
@@ -145,7 +145,44 @@
         </element>
         <element id="Observation.referenceRange">
             <path value="Observation.referenceRange" />
-            <max value="0" />
+            <short value="The reference range for a test group or profile." />
+            <definition value="The reference range element defines acceptable or expected ranges for a group lab observation result." />
+            <min value="0" />
+            <max value="*" />
+            <mustSupport value="true" />
+            <type>
+                <code value="BackboneElement" />
+            </type>
+        </element>
+    <element id="Observation.referenceRange.low">
+            <path value="Observation.referenceRange.low" />
+            <short value="Low boundary of the reference range" />
+            <definition value="Low limit for the reference range, if applicable." />
+            <min value="0" />
+            <max value="1" />
+            <type>
+                <code value="Quantity" />
+            </type>
+        </element>
+    <element id="Observation.referenceRange.high">
+            <path value="Observation.referenceRange.high" />
+            <short value="High boundary of the reference range" />
+            <definition value="High limit for the reference range, if applicable." />
+            <min value="0" />
+            <max value="1" />
+            <type>
+                <code value="Quantity" />
+            </type>
+        </element>
+     <element id="Observation.referenceRange.text">
+            <path value="Observation.referenceRange.text" />
+            <short value="Textual representation of the reference range" />
+            <definition value="Free-text representation for complex reference ranges, if needed." />
+            <min value="0" />
+            <max value="1" />
+            <type>
+                <code value="string" />
+            </type>
```

Comment:
@vickyjaiswal0308 Where did this come from? Did you use Forge or another software?

Reply:
I have not used any software or forge. Just edit this file in Notepad++ and took the reference from International FHIR - Observation Resource (https://hl7.org/fhir/R4/observation-definitions.html#Observation.referenceRange)

Reply:
Can you redo this using Forge as I don't expect to see this in a differential if there has been no changes to it. Thanks.